tests/platform_tests/api/test_chassis.py

```diff
@@ -400,7 +400,7 @@ def test_sfps(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost, platf
         except:
             pytest.fail("num_sfps is not an integer")
 
-        list_sfps = list(set(physical_port_indices))
```

Comment:
removing this might break Arista-7050cx3-32s-D48C8. @prgeor, why are we getting rid of this.

Reply:
'physical_port_indices' are already sorted and duplicate indices are already taken care:- [https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/api/test_chassis.py#L64](https://github.com/Azure/sonic-mgmt/blob/master/tests/platform_tests/api/test_chassis.py#L64)